src/main/kotlin/net/maizegenetics/phgv2/cli/StartServer.kt

```diff
@@ -21,17 +22,28 @@ import java.nio.file.Paths
  * If dbPath is supplied, we verify that it contains a valid hvcf_dataset.
  *
  */
-class StartServer : CliktCommand(help = "Starts PHGv2 BrAPI Server") {
+object StartServer : CliktCommand(help = "Starts PHGv2 BrAPI Server") {
 
     private val myLogger = LogManager.getLogger(StartServer::class.java)
+
     val dbPath by option(help = "Full path to folder where TileDB datasets are stored.  \nThis must be run at least once before starting the server. \nIf you have already run it for this server instance you do not need to supply this again.")
         .default("")
 
     val port by option(help = "The port on which the server will listen.  \nDefault is 8080.")
         .int()
         .default(8080)
 
+    lateinit var server: NettyApplicationEngine
+
+    fun serverURL(): String {
+        val actualPort = server.environment.connectors.map { it.port }.firstOrNull()
+        val host = server.environment.connectors.map { it.host }.firstOrNull() ?: "localhost"
+
+        return "http://$host:$actualPort/brapi/v2/"
+    }
```

Comment:
Wondering if this can be of help for a StopServer command.  User would have to be on the same host, and user would also have to know which port was used when starting the server.

not suggesting that for this branch - would be another branch later.

Reply:
Yes, possibly